themes/eclipse/composer.json

```
@@ -0,0 +1,10 @@
+{
+  "name": "mautic/theme-eclipse",
+  "description": "Eclipse theme",
+  "type": "mautic-theme",
+  "keywords": ["mautic","theme"],
+  "minimum-stability": "dev",
+  "require": {
+    "mautic/core-lib": "^5.0"
```

Update the version constraint for `mautic/core-lib`

---

I was thinking: this is also compatible with Mautic 4 afaik

All 5.x versions too
And also 6.x

what to do in this case?

---

If you are sure that it is compatible with Mautic versions starting from 4.0, you can set it as following:
```suggestion
    "mautic/core-lib": ">=4.0"
```
Just so someone would not ask in the Mautic forums about why the theme isn’t working on Mautic 4 etc. We won’t be able to prevent this later unless the PM deletes the release tag, which would create a mess.